x/concentrated-liquidity/types/migration.go

```diff
@@ -9,6 +9,44 @@ var MigratedIncentiveAccumulatorPoolIDs = map[uint64]struct{}{
 	1431: {},
 }
 
+var MigratedSpreadFactorAccumulatorPoolIDs = map[uint64]struct{}{
+	// token0 usdc
+	// token1 dai
+	// liquidity 55.8k
+	// https://app.osmosis.zone/pool/1260
+	1260: {},
+	// token0 eth
+	// token1 usdc
+	// liquidity 449k
+	// https://app.osmosis.zone/pool/1264
+	1264: {},
+	// token0 osmo
+	// token1 eth
+	// liquidity 395k
+	// https://app.osmosis.zone/pool/1477
+	1477: {},
+	// token0 osmo
+	// token1 eth
+	// liquidity 226k
+	// https://app.osmosis.zone/pool/1134
+	1134: {},
+	// token0 usdt
+	// token1 dai
+	// liquidity 4.85k
+	// https://app.osmosis.zone/pool/1261
+	1261: {},
+	// token0 eth
+	// token1 osmo
+	// liquidity 554k
+	// https://app.osmosis.zone/pool/1281
+	1281: {},
+	// token0 stdydx
+	// token1 dydx
+	// liquidity 418k
+	// https://app.osmosis.zone/pool/1423
+	1423: {},
```

Comment:
How were these selected?

It seems that most are 18 decimal pools. Are there no at-risk non-18 decimal pools or where only one of the tokens is an 18 decimal?

Reply:
This is every pool that has tripped the telemetry alert in the past few weeks, ordered by how often (first pool occurs most often, last pool occurs least often)

Reply:
Added a comment that explains this here https://github.com/osmosis-labs/osmosis/pull/7876/commits/0b77070a7ecafe73f9b0183b8730891395d57c22